src/main/java/graphql/introspection/Introspection.java

```diff
@@ -113,6 +115,20 @@ public static void addCodeForIntrospectionTypes(GraphQLCodeRegistry.Builder code
             return Assert.assertShouldNeverHappen("Unknown kind of type: %s", type);
         }
     };
+    public static final DataFetcher nameGraphQLElementDataFetcher = environment -> {
```

Comment:
why be public?  they dont need to be accessed for reuse do they?

Reply:
mh .. because all of them were public ... will change it